<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Steve Pederson" />

<meta name="date" content="2022-02-07" />

<title>QC</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ZR-75_H3K27ac_HiChIP_testrun</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="QC.html">QC</a>
</li>
<li>
  <a href="merged_qc.html">Merged QC</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">QC</h1>
<h4 class="author">Steve Pederson</h4>
<h4 class="date">2022-02-07</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-02-08
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>ZR-75_H3K27ac_HiChIP_testrun/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version 1.7.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220207code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220207)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220207code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220207)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomDRMCRLZR75H3K27acHiChIPtestruntreea17fa1fa23dc24999992e807217f61ca1caa3d84targetblanka17fa1fa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/DRMCRL/ZR-75_H3K27ac_HiChIP_testrun/tree/a17fa1fa23dc24999992e807217f61ca1caa3d84" target="_blank">a17fa1f</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomDRMCRLZR75H3K27acHiChIPtestruntreea17fa1fa23dc24999992e807217f61ca1caa3d84targetblanka17fa1fa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/DRMCRL/ZR-75_H3K27ac_HiChIP_testrun/tree/a17fa1fa23dc24999992e807217f61ca1caa3d84" target="_blank">a17fa1f</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/
    Ignored:    output/feather_output/21-01776_20220204_165523/21-01776.hic.input
    Ignored:    output/feather_output/21-01776_20220204_165523/21-01776.paired.bam
    Ignored:    output/feather_output/21-01776_20220204_165523/21-01776.paired.fixmated.bam
    Ignored:    output/feather_output/21-01776_20220204_165523/21-01776.paired.rmdup.bam
    Ignored:    output/feather_output/21-01776_20220204_165523/21-01776.paired.rmdup.bam.bai
    Ignored:    output/feather_output/21-01776_20220204_165523/21-01776.paired.srt.bam
    Ignored:    output/feather_output/21-01776_20220204_165523/21-01776.paired.srtn.rmdup.bam
    Ignored:    output/feather_output/21-01777_20220204_165809/21-01777.hic.input
    Ignored:    output/feather_output/21-01777_20220204_165809/21-01777.paired.bam
    Ignored:    output/feather_output/21-01777_20220204_165809/21-01777.paired.fixmated.bam
    Ignored:    output/feather_output/21-01777_20220204_165809/21-01777.paired.rmdup.bam
    Ignored:    output/feather_output/21-01777_20220204_165809/21-01777.paired.rmdup.bam.bai
    Ignored:    output/feather_output/21-01777_20220204_165809/21-01777.paired.srt.bam
    Ignored:    output/feather_output/21-01777_20220204_165809/21-01777.paired.srtn.rmdup.bam

Untracked files:
    Untracked:  analysis/QC.Rmd
    Untracked:  analysis/maps_setup.md
    Untracked:  analysis/merged_qc.Rmd
    Untracked:  code/21-01774_maps.sh
    Untracked:  code/21-01775_maps.sh
    Untracked:  code/21-01776_maps.sh
    Untracked:  code/21-01777_maps.sh
    Untracked:  code/E2_merged_maps.sh
    Untracked:  code/ED_merged_maps.sh
    Untracked:  code/maps.yaml
    Untracked:  data/GLL368_Arima-HiChIP_QC_Worksheet_A160170.xlsx
    Untracked:  data/ZR75_H3K27ac.bed
    Untracked:  data/fastq/
    Untracked:  output/21-01774.log
    Untracked:  output/21-01774_Arima_QC_deep.txt
    Untracked:  output/21-01774_Arima_QC_shallow.txt
    Untracked:  output/21-01775.log
    Untracked:  output/21-01775_Arima_QC_deep.txt
    Untracked:  output/21-01775_Arima_QC_shallow.txt
    Untracked:  output/21-01776.log
    Untracked:  output/21-01776_Arima_QC_deep.txt
    Untracked:  output/21-01776_Arima_QC_shallow.txt
    Untracked:  output/21-01777.log
    Untracked:  output/21-01777_Arima_QC_deep.txt
    Untracked:  output/21-01777_Arima_QC_shallow.txt
    Untracked:  output/E2_merged.log
    Untracked:  output/E2_merged_Arima_QC_deep.txt
    Untracked:  output/E2_merged_Arima_QC_shallow.txt
    Untracked:  output/ED_merged.log
    Untracked:  output/ED_merged_Arima_QC_deep.txt
    Untracked:  output/ED_merged_Arima_QC_shallow.txt
    Untracked:  output/MAPS_output/21-01774_20220207_135015/
    Untracked:  output/MAPS_output/21-01774_20220208_130435/
    Untracked:  output/MAPS_output/21-01774_20220208_164744/
    Untracked:  output/MAPS_output/21-01775_20220207_142027/
    Untracked:  output/MAPS_output/21-01775_20220208_130439/
    Untracked:  output/MAPS_output/21-01775_20220208_164753/
    Untracked:  output/MAPS_output/21-01776_20220208_130443/
    Untracked:  output/MAPS_output/21-01776_20220208_132152/
    Untracked:  output/MAPS_output/21-01776_20220208_132255/
    Untracked:  output/MAPS_output/21-01776_20220208_132311/
    Untracked:  output/MAPS_output/21-01776_20220208_132327/
    Untracked:  output/MAPS_output/21-01776_20220208_132443/
    Untracked:  output/MAPS_output/21-01777_20220208_130445/
    Untracked:  output/MAPS_output/21-01777_20220208_133340/
    Untracked:  output/MAPS_output/E2_merged_20220208_173413/
    Untracked:  output/MAPS_output/E2_merged_current
    Untracked:  output/MAPS_output/ED_merged_20220208_173429/
    Untracked:  output/MAPS_output/ED_merged_current
    Untracked:  output/arcplot_and_metaplot/E2_merged.coverage.bigwig
    Untracked:  output/arcplot_and_metaplot/E2_merged.coverage_matrix.tab.gz
    Untracked:  output/arcplot_and_metaplot/E2_merged.heatmap.pdf
    Untracked:  output/arcplot_and_metaplot/ED_merged.coverage.bigwig
    Untracked:  output/arcplot_and_metaplot/ED_merged.coverage_matrix.tab.gz
    Untracked:  output/arcplot_and_metaplot/ED_merged.heatmap.pdf
    Untracked:  output/feather_output/21-01774_20220207_135015/
    Untracked:  output/feather_output/21-01774_20220208_130435/
    Untracked:  output/feather_output/21-01774_20220208_164744/
    Untracked:  output/feather_output/21-01775_20220207_142027/
    Untracked:  output/feather_output/21-01775_20220208_130439/
    Untracked:  output/feather_output/21-01775_20220208_164753/
    Untracked:  output/feather_output/21-01776_20220208_130443/
    Untracked:  output/feather_output/21-01776_20220208_132152/
    Untracked:  output/feather_output/21-01776_20220208_132255/
    Untracked:  output/feather_output/21-01776_20220208_132311/
    Untracked:  output/feather_output/21-01776_20220208_132327/
    Untracked:  output/feather_output/21-01776_20220208_132443/
    Untracked:  output/feather_output/21-01777_20220208_130445/
    Untracked:  output/feather_output/21-01777_20220208_133340/
    Untracked:  output/feather_output/E2_merged_20220208_173413/
    Untracked:  output/feather_output/E2_merged_current
    Untracked:  output/feather_output/ED_merged_20220208_173429/
    Untracked:  output/feather_output/ED_merged_current

Unstaged changes:
    Modified:   .gitignore
    Deleted:    README.md
    Deleted:    ZR75_H3K27ac.bed
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Modified:   analysis/index.Rmd
    Deleted:    analysis/license.Rmd
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01774/CGATGT/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01774/CGATGT/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01774/all/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01774/all/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01775/TGACCA/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01775/TGACCA/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01775/all/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01775/all/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01776/ACAGTG/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01776/ACAGTG/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01776/all/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01776/all/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01777/GCCAAT/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01777/GCCAAT/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01777/all/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/21-01777/all/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/all/all/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/SAGCQA0256_GeriLavenLaw_02122021/all/all/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/all/all/all/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/all/all/all/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/default/Undetermined/all/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/default/Undetermined/all/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/default/Undetermined/unknown/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/default/Undetermined/unknown/laneBarcode.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/default/all/all/lane.html
    Deleted:    fastq/Reports/html/000000000-GB6P6/default/all/all/laneBarcode.html
    Deleted:    fastq/Reports/html/Report.css
    Deleted:    fastq/Reports/html/index.html
    Deleted:    fastq/Reports/html/tree.html
    Deleted:    fastq/Stats/AdapterTrimming.txt
    Deleted:    fastq/Stats/ConversionStats.xml
    Deleted:    fastq/Stats/DemultiplexingStats.xml
    Deleted:    fastq/Stats/DemuxSummaryF1L1.txt
    Deleted:    fastq/Stats/FastqSummaryF1L1.txt
    Deleted:    fastq/Stats/Stats.json
    Deleted:    fastq/fastq.md5
    Deleted:    maps.yaml
    Deleted:    output/MAPS_output/21-01774_20220204_164412/execution_script_copy
    Deleted:    output/MAPS_output/21-01774_20220204_164412/maps_21-01774.maps
    Modified:   output/MAPS_output/21-01774_current
    Deleted:    output/MAPS_output/21-01775_20220204_165131/execution_script_copy
    Deleted:    output/MAPS_output/21-01775_20220204_165131/maps_21-01775.maps
    Modified:   output/MAPS_output/21-01775_current
    Modified:   output/MAPS_output/21-01776_current
    Modified:   output/MAPS_output/21-01777_current
    Modified:   output/arcplot_and_metaplot/21-01774.coverage_matrix.tab.gz
    Modified:   output/arcplot_and_metaplot/21-01774.heatmap.pdf
    Modified:   output/arcplot_and_metaplot/21-01775.coverage_matrix.tab.gz
    Modified:   output/arcplot_and_metaplot/21-01775.heatmap.pdf
    Modified:   output/arcplot_and_metaplot/21-01776.coverage_matrix.tab.gz
    Modified:   output/arcplot_and_metaplot/21-01776.heatmap.pdf
    Modified:   output/arcplot_and_metaplot/21-01777.coverage_matrix.tab.gz
    Modified:   output/arcplot_and_metaplot/21-01777.heatmap.pdf
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr1.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr1.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr10.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr10.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr11.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr11.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr12.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr12.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr13.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr13.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr14.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr14.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr15.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr15.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr16.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr16.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr17.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr17.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr18.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr18.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr19.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr19.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr2.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr2.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr20.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr20.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr21.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr21.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr22.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr22.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr3.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr3.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr4.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr4.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr5.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr5.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr6.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr6.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr7.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr7.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr8.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr8.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr9.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chr9.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chrX.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chrX.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chrY.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.chrY.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.feather.log
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.feather.qc
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.feather.qc.tsv
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.long.intra.bedpe
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.paired.fixmated.markdup.stats
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.paired.rmdup.flagstat
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.paired.srt.bam.flagstat
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.qc
    Deleted:    output/feather_output/21-01774_20220204_164412/21-01774.shrt.vip.bed
    Deleted:    output/feather_output/21-01774_20220204_164412/execution_script_copy
    Modified:   output/feather_output/21-01774_current
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr1.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr1.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr10.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr10.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr11.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr11.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr12.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr12.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr13.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr13.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr14.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr14.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr15.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr15.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr16.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr16.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr17.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr17.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr18.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr18.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr19.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr19.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr2.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr2.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr20.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr20.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr21.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr21.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr22.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr22.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr3.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr3.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr4.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr4.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr5.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr5.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr6.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr6.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr7.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr7.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr8.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr8.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr9.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chr9.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chrX.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chrX.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.chrY.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.feather.log
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.feather.qc
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.feather.qc.tsv
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.long.intra.bedpe
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.paired.fixmated.markdup.stats
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.paired.rmdup.flagstat
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.paired.srt.bam.flagstat
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.qc
    Deleted:    output/feather_output/21-01775_20220204_165131/21-01775.shrt.vip.bed
    Deleted:    output/feather_output/21-01775_20220204_165131/execution_script_copy
    Modified:   output/feather_output/21-01775_current
    Modified:   output/feather_output/21-01776_current
    Modified:   output/feather_output/21-01777_current

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<pre class="r"><code>library(tidyverse)
library(readxl)
library(rjson)
library(pander)
library(glue)</code></pre>
<pre class="r"><code>panderOptions(&quot;big.mark&quot;, &quot;,&quot;)
theme_set(theme_bw())</code></pre>
<pre class="r"><code>samples &lt;- fromJSON(file = here::here(&quot;data/fastq/Stats/Stats.json&quot;))$ConversionResults %&gt;% 
  .[[1]] %&gt;% 
  .[[&quot;DemuxResults&quot;]] %&gt;% 
  lapply(as_tibble) %&gt;% 
  bind_rows() %&gt;% 
  mutate(
    Index = vapply(IndexMetrics, function(x) x$IndexSequence, character(1))
  ) %&gt;% 
  distinct(
    SampleId, Index, NumberReads, Yield
  ) %&gt;% 
  left_join(
    here::here(&quot;data/GLL368_Arima-HiChIP_QC_Worksheet_A160170.xlsx&quot;) %&gt;%
      read_excel(sheet = &quot;Library Complexity QC&quot;, skip = 75) %&gt;% 
      mutate(
        treat = str_extract(Sample, &quot;E.&quot;) %&gt;% str_trim(),
        replicate = str_remove_all(Sample, &quot;.+rep &quot;)
      ) %&gt;% 
      dplyr::select(Sample, Index = `i7 index sequence`, treat, replicate),
    by = &quot;Index&quot;
  ) %&gt;% 
  dplyr::select(Sample, everything())</code></pre>
<pre class="r"><code>samples %&gt;% 
  dplyr::select(Sample, SampleId, NumberReads, Index) %&gt;% 
  pander(
    caption = &quot;Library sizes for each sample&quot;
  )</code></pre>
<table style="width:79%;">
<caption>Library sizes for each sample</caption>
<colgroup>
<col width="31%" />
<col width="15%" />
<col width="19%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Sample</th>
<th align="center">SampleId</th>
<th align="center">NumberReads</th>
<th align="center">Index</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">ZR-75-1 24h E rep 1</td>
<td align="center">21-01774</td>
<td align="center">690,614</td>
<td align="center">CGATGT</td>
</tr>
<tr class="even">
<td align="center">ZR-75-1 24h E rep 2</td>
<td align="center">21-01775</td>
<td align="center">567,918</td>
<td align="center">TGACCA</td>
</tr>
<tr class="odd">
<td align="center">ZR-75-1 24h ED rep 1</td>
<td align="center">21-01776</td>
<td align="center">601,342</td>
<td align="center">ACAGTG</td>
</tr>
<tr class="even">
<td align="center">ZR-75-1 24h ED rep 2</td>
<td align="center">21-01777</td>
<td align="center">642,546</td>
<td align="center">GCCAAT</td>
</tr>
</tbody>
</table>
</div>
<div id="maps-pipeline" class="section level1 tabset">
<h1 class="tabset">MAPS Pipeline</h1>
<pre class="r"><code>bed_peaks &lt;- here::here(&quot;data/ZR75_H3K27ac.bed&quot;)
in_path &lt;- here::here(&quot;data/fastq&quot;)
out_path &lt;- here::here(&quot;output&quot;)
ref_path &lt;- &quot;/refs/gencode/grch37/GRCh37.primary_assembly.genome.fa&quot;
maps_path &lt;- &quot;/opt/MAPS/bin/Arima-MAPS_v2.0.sh&quot;
samples$SampleId %&gt;% 
  lapply(
    function(x) {
      glue(
        &quot;
        #! /bin/bash
        
        ## This script will need to br run within the MAPS coda environment
        
        {maps_path} \\
          -C 0 \\
          -m {bed_peaks} \\
          -I {in_path}/{x} \\
          -O {out_path}  \\
          -o &#39;hg19&#39; \\
          -b  {ref_path} \\
          -t 8 \\
          -f 0 &gt; {out_path}/{x}.log
          
        &quot;
      ) %&gt;% 
        write_lines(
          here::here(&quot;code&quot;, glue(&quot;{x}_maps.sh&quot;))
        )
    }
  )</code></pre>
<p>The MAPS pipeline was run on individual samples. In two cases, multiple errors were returned and the expected output as specified <a href="https://github.com/ijuric/MAPS/tree/master/Arima_Genomics#output-directorysample-name_arima_qc_shallowtxt">here</a> was not returned. However, the pipeline completed successfully for <em>21-01774</em>, <em>21-01775</em>, <em>21-01776</em>, <em>21-01777</em>, <em>E2_merged</em> and <em>ED_merged</em></p>
<pre class="r"><code>shallow_df &lt;- here::here(&quot;output&quot;, glue(&quot;{samples$SampleId}_Arima_QC_shallow.txt&quot;)) %&gt;% 
  lapply(read_tsv) %&gt;% 
  bind_rows()</code></pre>
<p>The recommended Target Raw PE Reads are.</p>
<pre class="r"><code>shallow_df %&gt;% 
  left_join(samples, by = c(&quot;Sample Name&quot; = &quot;SampleId&quot;)) %&gt;% 
  dplyr::select(`Sample Name`, treat, `Target Raw PE Reads`) %&gt;% 
  mutate(`Target Raw PE Reads` = `Target Raw PE Reads` / 1e6) %&gt;% 
  pander(
    caption = &quot;Target sequencing depth for each sample (millions of reads). This appears unrealistic.&quot;
  )</code></pre>
<table style="width:61%;">
<caption>Target sequencing depth for each sample (millions of reads). This appears unrealistic.</caption>
<colgroup>
<col width="19%" />
<col width="11%" />
<col width="30%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Sample Name</th>
<th align="center">treat</th>
<th align="center">Target Raw PE Reads</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">21-01774</td>
<td align="center">E</td>
<td align="center">1,040</td>
</tr>
<tr class="even">
<td align="center">21-01775</td>
<td align="center">E</td>
<td align="center">1,030</td>
</tr>
<tr class="odd">
<td align="center">21-01776</td>
<td align="center">ED</td>
<td align="center">1,020</td>
</tr>
<tr class="even">
<td align="center">21-01777</td>
<td align="center">ED</td>
<td align="center">1,060</td>
</tr>
</tbody>
</table>
<pre class="r"><code>shallow_df %&gt;% 
  left_join(samples, by = c(&quot;Sample Name&quot; = &quot;SampleId&quot;)) %&gt;% 
  dplyr::select(`Sample Name`, treat, starts_with(&quot;%&quot;), -ends_with(&quot;Dups&quot;), -contains(&quot;Mapped&quot;)) %&gt;% 
  pivot_longer(cols= starts_with(&quot;%&quot;)) %&gt;% 
  ggplot(aes(`Sample Name`, value, fill = treat)) +
  geom_col() +
  facet_wrap(~name)</code></pre>
<div class="figure" style="text-align: center">
<img src="figure/QC.Rmd/plot_percent-1.png" alt="% or interactions pairs in the categories provided by MAPS output" width="672" />
<p class="caption">
% or interactions pairs in the categories provided by MAPS output
</p>
</div>
<div id="peak-coverage" class="section level2">
<h2>Peak Coverage</h2>
<pre class="r"><code>samples$SampleId %&gt;% 
  lapply(
    function(x) {
      pdftools::pdf_convert(
        pdf = here::here(glue(&quot;output/arcplot_and_metaplot/{x}.heatmap.pdf&quot;)), 
        pages = 1, 
        format = &quot;png&quot;, 
        filenames = here::here(glue(&quot;docs/assets/{x}.heatmap.png&quot;)), 
        dpi = 150
      )
    }
  )</code></pre>
<p>Pre-defined H3K27ac peaks were supplied by combining consensus peaks across both E2 and E2+DHT from previous experiments.</p>
<p><img src="assets/21-01774.heatmap.png" style="width:23.0%;height:100.0%" alt="21-01774" /> <img src="assets/21-01775.heatmap.png" style="width:23.0%;height:100.0%" alt="21-01775" /> <img src="assets/21-01776.heatmap.png" style="width:23.0%;height:100.0%" alt="21-01776" /> <img src="assets/21-01777.heatmap.png" style="width:23.0%;height:100.0%" alt="21-01777" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.1 (2021-08-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.3 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0

locale:
 [1] LC_CTYPE=en_AU.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_AU.UTF-8        LC_COLLATE=en_AU.UTF-8    
 [5] LC_MONETARY=en_AU.UTF-8    LC_MESSAGES=en_AU.UTF-8   
 [7] LC_PAPER=en_AU.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_AU.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] glue_1.6.1      pander_0.6.4    rjson_0.2.21    readxl_1.3.1   
 [5] forcats_0.5.1   stringr_1.4.0   dplyr_1.0.7     purrr_0.3.4    
 [9] readr_2.1.2     tidyr_1.2.0     tibble_3.1.6    ggplot2_3.3.5  
[13] tidyverse_1.3.1 workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] httr_1.4.2       sass_0.4.0       bit64_4.0.5      vroom_1.5.7     
 [5] jsonlite_1.7.3   here_1.0.1       modelr_0.1.8     bslib_0.3.1     
 [9] assertthat_0.2.1 getPass_0.2-2    highr_0.9        cellranger_1.1.0
[13] yaml_2.2.2       pillar_1.7.0     backports_1.4.1  digest_0.6.29   
[17] promises_1.2.0.1 rvest_1.0.2      colorspace_2.0-2 htmltools_0.5.2 
[21] httpuv_1.6.5     pkgconfig_2.0.3  broom_0.7.12     haven_2.4.3     
[25] scales_1.1.1     processx_3.5.2   whisker_0.4      later_1.3.0     
[29] tzdb_0.2.0       git2r_0.29.0     generics_0.1.2   farver_2.1.0    
[33] ellipsis_0.3.2   withr_2.4.3      cli_3.1.1        magrittr_2.0.2  
[37] crayon_1.4.2     evaluate_0.14    ps_1.6.0         fs_1.5.2        
[41] fansi_1.0.2      xml2_1.3.3       tools_4.1.1      hms_1.1.1       
[45] lifecycle_1.0.1  munsell_0.5.0    reprex_2.0.1     callr_3.7.0     
[49] compiler_4.1.1   jquerylib_0.1.4  rlang_1.0.1      grid_4.1.1      
[53] rstudioapi_0.13  labeling_0.4.2   rmarkdown_2.11   gtable_0.3.0    
[57] DBI_1.1.2        R6_2.5.1         lubridate_1.8.0  knitr_1.37      
[61] fastmap_1.1.0    bit_4.0.4        utf8_1.2.2       rprojroot_2.0.2 
[65] stringi_1.7.6    parallel_4.1.1   Rcpp_1.0.8       vctrs_0.3.8     
[69] dbplyr_2.1.1     tidyselect_1.1.1 xfun_0.29       </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
