---
title: "merged_qc"
author: "Steve Pederson"
date: "2022-02-08"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---

```{r opts, echo=FALSE}
knitr::opts_chunk$set(
  message = FALSE, warning = FALSE
)
```



## Introduction

```{r pkgs}
library(tidyverse)
library(readxl)
library(rjson)
library(pander)
library(glue)
```


```{r options}
panderOptions("big.mark", ",")
theme_set(theme_bw())
```

This workflow merges the samples within the same treatment groups checking to see if:

- Merging with increased depth allows completion of the pipeline
- Predictions of sequencing depth remain consistent

# MAPS Pipeline {.tabset}

```{r write_bash_scripts, eval=FALSE}
bed_peaks <- here::here("data/ZR75_H3K27ac.bed")
in_path <- here::here("data/fastq")
out_path <- here::here("output")
ref_path <- "/refs/gencode/grch37/GRCh37.primary_assembly.genome.fa"
maps_path <- "/opt/MAPS/bin/Arima-MAPS_v2.0.sh"
paste0(c("E2", "ED"), "_merged") %>% 
  lapply(
    function(x) {
      glue(
        "
        #! /bin/bash
        
        ## This script will need to br run within the MAPS conda environment
        
        {maps_path} \\
          -C 0 \\
          -m {bed_peaks} \\
          -I {in_path}/{x} \\
          -O {out_path}  \\
          -o 'hg19' \\
          -b  {ref_path} \\
          -t 8 \\
          -f 0 > {out_path}/{x}.log
          
        "
      ) %>% 
        write_lines(
          here::here("code", glue("{x}_maps.sh"))
        )
    }
  )
```


```{r shallow_df}
shallow_df <- here::here("output", paste0(c("E2", "ED"),  "_merged_Arima_QC_shallow.txt")) %>% 
  lapply(read_tsv) %>% 
  bind_rows()
```

The recommended Target Raw PE Reads are.

```{r target_tab}
shallow_df %>% 
  dplyr::select(`Sample Name`, `Target Raw PE Reads`) %>% 
  mutate(`Target Raw PE Reads` = `Target Raw PE Reads` / 1e6) %>% 
  pander(
    caption = "Target sequencing depth for each sample (millions of reads). This appears unrealistic."
  )
```


```{r plot_percent, fig.cap = "% or interactions pairs in the categories provided by MAPS output"}
shallow_df %>% 
  dplyr::select(`Sample Name`, starts_with("%"), -ends_with("Dups"), -contains("Mapped")) %>% 
  pivot_longer(cols= starts_with("%")) %>% 
  ggplot(aes(`Sample Name`, value, fill = `Sample Name`)) +
  geom_col() +
  facet_wrap(~name)
```


## Peak Coverage

```{r convert_heatmaps, eval = FALSE}
c("E2_merged", "ED_merged") %>% 
  lapply(
    function(x) {
      pdftools::pdf_convert(
        pdf = here::here(glue("output/arcplot_and_metaplot/{x}.heatmap.pdf")), 
        pages = 1, 
        format = "png", 
        filenames = here::here(glue("docs/assets/{x}.heatmap.png")), 
        dpi = 150
      )
    }
  )
```


Pre-defined H3K27ac peaks were supplied by combining consensus peaks across both E2 and E2+DHT from previous experiments.

![E2_merged](`r file.path("assets", "E2_merged.heatmap.png")`){width=30% height=100%}
![ED_merged](`r file.path("assets", "ED_merged.heatmap.png")`){width=30% height=100%}

